src/components/contact/SocialMedia.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebook, faTwitter, faInstagram } from "@fortawesome/free-brands-svg-icons";
+
+import styles from './assets/socialMedia.scss';
+
+const SocialMedia = () => {
+  return (
+    <div className={styles.socialMedia}>
+      <div className={styles.Icon__Wrapper}>
+        <a target="_blank" href={process.env.REACT_APP_FB_URL}>
+          <FontAwesomeIcon icon={faFacebook} size='2x' className={styles.greenIcons} />
```

Comment:
Should this be wrapped in an icon component and that decided whether to use font awesome or not.. what if there are a set of material icons we need to use later?